tests/components/ps4/test_config_flow.py

```diff
@@ -65,6 +82,110 @@
         assert result['data']['devices'] == [MOCK_DEVICE]
         assert result['title'] == MOCK_TITLE
 
+    # Add entry using result data.
+    mock_data = {
+        CONF_TOKEN: result['data'][CONF_TOKEN],
+        'devices': result['data']['devices']}
+    entry = MockConfigEntry(domain=ps4.DOMAIN, data=mock_data)
+    entry.add_to_manager(manager)
+
+    # Check if entry exists.
+    assert len(manager.async_entries()) == 1
+    # Check if there is a device config in entry.
+    assert len(entry.data['devices']) == 1
+
+
+async def test_multiple_flow_implementation(hass):
+    """Test multiple device flows."""
+    flow = ps4.PlayStation4FlowHandler()
+    flow.hass = hass
+
+    # Init config manager.
+    manager = config_entries.ConfigEntries(hass, {})
+    manager._entries = []   # noqa: pylint: disable=protected-access
```

Comment:
Why do we need to access a protected attribute on the manager? That doesn't look right.

Reply:
Ok Ill just use MockConfigEntry then.

Reply:
Removed.  @MartinHjelmare